docs/install_guides/configurations-for-intel-gpu.md

```diff
@@ -46,11 +41,6 @@ You've completed all required configuration steps to perform inference on proces
 
 ## Windows
 
-This section will help you check if you require driver installation. Install indicated version or higher.
-
-If your applications offload computation to **Intel® Integrated Graphics**, you must have the Intel Graphics Driver for Windows installed on your hardware.
-[Download and install the recommended version](https://downloadcenter.intel.com/download/30079/Intel-Graphics-Windows-10-DCH-Drivers). 
-
 To check if you have this driver installed:
```

Comment:
I do not understand flow here. Instruction suggests to check which driver I have but what if I'm not? Where should I go?

Reply:
Is it possible to have a Windows install without GPU drivers? GPU inference has always worked out of the box for me (on computers with an Intel iGPU). Could we have a sentence that GPU drivers are included in Windows and that it is not usually necessary to install anything, and then maybe add that if you want to use the validated version, do X?

Reply:
> Is it possible to have a Windows install without GPU drivers? GPU inference has always worked out of the box for me (on computers with an Intel iGPU). Could we have a sentence that GPU drivers are included in Windows and that it is not usually necessary to install anything, and then maybe add that if you want to use the validated version, do X?

I believe it works due to automatic updates which might not be the case everywhere. Also not sure how systems with iGPU and dGPU (of whatever vendor) are used and how updates should be handled there. So for me it worth mentioning how to install this.